radconpy/radcon.py

```diff
@@ -70,6 +137,26 @@ def _on_disconnected(self):
         self._reconnect_thread = threading.Thread(target=self._reconnect)
         self._reconnect_thread.start()
 
+    def _on_data(self, timestamp, hardware_timestamp, pulse_length):
+        if len(self._pulse_timestamps) == 0:
+            self._pulse_next_update = timestamp + timedelta(seconds=self._cpm_window_width)
+
+        if len(self._pulse_next_update) > 1000:
```

Comment:
```suggestion
        if len(self._pulse_timestamps) > 1000:
```
@pfgryz Shouldn't this be _pulse_timestamps instead of _pulse_next_update?